chore: add a tip to run the program in paranoid mode (#5)
Bearer/scanner-poc: README.md

Run the binary passing the list of repository folders

```console
$ ~/.bearer/bearer-cli local <path_to_source_code_root_folder_1> <path_to_source_code_root_folder_2>
$ ls
bearer-cli.zip
```

#### Paranoid mode

<details>
The following method is less practical to use and has more limitations;
However, it can help you build an isolated environment to run the scanner.
	
```console
$ docker run -it --rm \
	-w "$PWD" -v "$PWD:$PWD" \
	buildpack-deps sh -ec \
	'curl -sL "https://raw.githubusercontent.com/Bearer/scanner-poc/main/download.sh" | bash -s && ~/.bearer/bearer-cli local <path_to_source_code_root_folder_1> <path_to_source_code_root_folder_2>'
[...]
$ ls bearer_report_*.zip
bearer_report_2020-11-12-144452.zip
```

You can go even further by building an image that contains `git` and the binary, then run docker with the `--network=none` flag.

> _disclaimer: this method is only shown as an example, please use the standard version if you want to get support from the Bearer team._
> and can't be supported by the Bearer; It is very limited (only supporting forked repos) and has more chance of 

Reviewer: ```suggestion
_disclaimer: this method is only shown as an example. Please use the standard version if you want to get support from the Bearer team._
It is very limited (only supporting forked repositories/projects).
```

Author: thx 🙏
